src/app/components/DappBanner/index.tsx

```diff
@@ -0,0 +1,73 @@
+import { FC } from 'react'
+import Card from '@mui/material/Card'
+import CardContent from '@mui/material/CardContent'
+import Typography from '@mui/material/Typography'
+import { COLORS } from '../../../styles/theme/colors'
+import Box from '@mui/material/Box'
+import { useTranslation } from 'react-i18next'
+import { SearchScope } from '../../../types/searchScope'
+import { getDappForAddress } from '../../config/dapps'
+import Button from '@mui/material/Button'
+import { useScreenSize } from '../../hooks/useScreensize'
+
+export const DappBanner: FC<{ scope: SearchScope; address: string | undefined }> = ({ scope, address }) => {
```

Comment:
Add a comment or rename address parameter to clarify that it's eth address
(related https://github.com/oasisprotocol/explorer/issues/711)

Reply:
OK, renamed to `ethAddress`. (Also renamed the function to `getDappForEthAddress()`.)